util/include/Timers.hpp

```diff
@@ -40,11 +48,14 @@ class Timers {
     };
 
    private:
-    Timer(std::chrono::milliseconds d, Type t, std::function<void()> cb) {
+    Timer(std::chrono::milliseconds d, Type t, std::function<void(Handle)> cb) {
       Timer(d, t, cb, std::chrono::steady_clock::now());
     }
```

Comment:
Don't we need to handle stop/remove timers differently now?

Reply:
We never stop or remove timers in the current code AFAIK. Recurring timers can be stopped and removed via `timers_.cancel(handle)`